__test__/pet.test.js

```diff
@@ -0,0 +1,96 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Ziggy')).toBeInstanceOf(Object);
+    });
+    it('sets the name property', () => {
+      const pet = new Pet('Ziggy');
+      expect(pet.name).toEqual('Ziggy')
+    });
+    it('has a initial age of 0', () => {
+      const pet = new Pet('Ziggy');
+      expect(pet.age).toEqual(0);
+    });
+    it('has initial hunger of 0', () => {
+      const pet = new Pet('Ziggy')
+      expect(pet.hunger).toEqual(0);
+    });
+    it('has initial fitness of 10', () => {
+      const pet = new Pet('Ziggy')
+      expect(pet.fitness).toEqual(10);
+    });
+  });
+  
+  describe('growUp', () => {
```

Comment:
There is a problem from here to the end of the file: all the describe blocks after this one are located _inside_ this one, and will therefore appear as a subset of tests inside the grow up function (If you run the tests now, do the `walk`, `feed` and `checkup` functions appear intended below `growUp`?)
This describe block should be closed off and the following ones separated as the first describe block is.

Regards the `growUp` block itself, is has the same issues with indentation in the bodies of the `it` blocks as above